WordPress/src/main/java/org/wordpress/android/util/EncryptionUtils.java

```diff
@@ -0,0 +1,116 @@
+package org.wordpress.android.util;
+
+import android.util.Base64;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.libsodium.jni.NaCl;
+
+public class EncryptionUtils {
+    static final int BOX_SEALBYTES = NaCl.sodium().crypto_box_sealbytes();
+    static final int XCHACHA20POLY1305_KEYBYTES = NaCl.sodium().crypto_secretstream_xchacha20poly1305_keybytes();
+    static final int XCHACHA20POLY1305_STATEBYTES = NaCl.sodium().crypto_secretstream_xchacha20poly1305_statebytes();
+    static final int XCHACHA20POLY1305_HEADERBYTES = NaCl.sodium().crypto_secretstream_xchacha20poly1305_headerbytes();
+    static final int XCHACHA20POLY1305_ABYTES = NaCl.sodium().crypto_secretstream_xchacha20poly1305_abytes();
+    static final int BASE64_ENCODE_FLAGS = Base64.DEFAULT;
+
+    static final short XCHACHA20POLY1305_TAG_FINAL = 
+            (short) NaCl.sodium().crypto_secretstream_xchacha20poly1305_tag_final();
+    static final short XCHACHA20POLY1305_TAG_0 = (short) 0;
```

Comment:
This shouldn't be a magic number – it has its own tag name (and should have an associated symbol you can call): `crypto_secretstream_xchacha20poly1305_TAG_MESSAGE`

Reply:
Aw man! I totally missed that.  Thank you for pointing that out. 😅

Reply:
now using the tag name with this change: 09c15a87bc1be23431b4f78d532b5b72837d780c